cloud-aws/src/main/resources/templates/aws-cf-dbstack.ftl

```diff
@@ -87,7 +97,42 @@
   },
 
   "Resources" : {
-
+  <#if hasSecurityGroup>
+  <#else>
+        "VPCSecurityGroup": {
+          "Type" : "AWS::EC2::SecurityGroup",
+          "Properties" : {
+              "GroupDescription" : "Security group attached to the database server",
+              "GroupName" : { "Ref":"DBSecurityGroupNameParameter"},
+              "SecurityGroupEgress" : [
+                    {
+                      "CidrIp": "0.0.0.0/0",
+                      "IpProtocol": "-1"
+                    }
+              ],
+              "SecurityGroupIngress" : [
+                    {
+                      "IpProtocol" : "tcp",
+                      "FromPort": 5432,
+                      "ToPort" : 5432,
+                      "CidrIp" : "0.0.0.0/0"
```

Comment:
This is too wide open - it will allow access from the internet. Instead, access should be restricted to every subnet in the VPC.

(Same for port 22, which probably isn't necessary anyway but shouldn't hurt.)

Reply:
Ok, I added the VPC cidr, as discussed. Later we should address if this is good enough (change env can change the network).